Userland/Libraries/LibCore/Socket.h

```diff
@@ -329,6 +329,11 @@ class LocalSocket final : public Socket {
 
     ErrorOr<int> receive_fd(int flags);
     ErrorOr<void> send_fd(int fd);
+
+    static constexpr size_t MAX_TRANSFER_FDS = 64;
```

Comment:
nit: if this isn't needed outside of the below methods, I'd just define this locally in Socket.cpp. Much less of a rebuild if we change the value later on.

Reply:
moved to cpp